package.json

```diff
@@ -9,11 +9,11 @@
     "test": "aegir test",
     "test:node": "aegir test -t node",
     "test:browser": "aegir test -t browser",
-    "build": "aegir build",
+    "build": "aegir build --no-types",
```

Comment:
Don't we want types?

Reply:
we want types, but we are not adding types to all libp2p modules for the time being: https://github.com/libp2p/js-libp2p/issues/659

We are focusing first on guarantee that libp2p provides types for the user surface area, which is essentially libp2p + libp2p-interfaces. Slowly, we should roll up the remaining modules, as some libp2p modules (including this) also need libp2p and libp2p-interfaces to have types, as libp2p is provided to the modules.

Reply:
Either way, once I release interfaces I can add types here as this should be super fast